lib/pure/collections/tables.nim

```diff
@@ -2893,15 +2893,15 @@ iterator mvalues*[A](t: CountTableRef[A]): var int =
 
 proc hash*[K,V](s: Table[K,V]): Hash =
   for p in pairs(s):
-    result = result xor hash(p)
+    result = result !& hash(p)
```

Comment:
This is definitely not right. `!&` is order-dependent, but you need something order-independent. `xor` fits that bill, but `+` is probably better, since identical values don't cancel.

Reply:
No idea why `Hash` was decided to be `int` instead of `uint`, but you're right. In that case, I'd go with `+%` instead of `+`